Makefile

```diff
@@ -21,6 +21,7 @@ FLAVORS ?= \
 	pacific,centos,8 \
 	quincy,centos,8 \
 	reef,centos,8 \
+	squid,centos,8 \
```

Comment:
I remember when we branched reef I was told squid would be built on el9-based images, are we going to continue with el8 at the end?

Reply:
I don't know, so please let me loop @cbodley in.

Reply:
@guits we're still planning to drop el8 for squid, it's just been a slog to actually get el9 container builds working. i'll cc you on a discussion in the #sepia slack where dmick identified some missing packages need by the container build

Reply:
Would it be fine to merge as is, with squid-on-el8, and later – when we're ready – switch squid to to el9?

Reply:
absolutely, this shouldn't block the pr

there are separate discussions about the fate of this ceph-container repo in https://github.com/ceph/ceph-container/issues/2171 and moving the container builds into the ceph repo with https://github.com/ceph/ceph/pull/54575. hopefully that can happen for squid, but someone will need to take the initiative on that

Reply:
Sounds good, thanks @cbodley for clarifying.

Reply:
Mind linking to that Slack thread here?

Reply:
never tried to link a thread before - hope this works? https://ceph-storage.slack.com/archives/C1HFJ4VTN/p1701386424620719